cmd/facedetect-from-url/main.go

```diff
@@ -0,0 +1,79 @@
+// What it does:
+//
+// This example uses the CascadeClassifier class to detect faces from url,
+// and draw a rectangle around each of them, before displaying them within a Window.
+//
+// How to run:
+//
+// facedetect-from-url [image URL] [classifier XML file] [image file]
+//
+// 		go run ./cmd/facedetect/main.go https://raw.githubusercontent.com/hybridgroup/gocv/master/images/face.jpg data/haarcascade_frontalface_default.xml output.jpg
```

Comment:
This line needs to be updated to correctly point to the location of this new example, and to use the correct syntax.